components/inputs/input-text.js

```diff
@@ -77,10 +77,6 @@ class InputText extends FormElementMixin(RtlMixin(LitElement)) {
 			 * Imposes a lower character limit
 			 */
 			minlength: { type: Number },
-			/**
-			 * Name of the input
-			 */
-			name: { type: String },
```

Comment:
@krshnpatel-d2l FYI you'll want to remove `name` from numeric input as you'll get it for free now from the FormElementMixin.